core/domain/classifier_domain.py

```diff
@@ -125,46 +125,123 @@ def __init__(self, job_id, algorithm_id, interaction_id, exp_id,
 
     @property
     def job_id(self):
+        """Returns the job_id of the classifier training job.
+
+        Returns:
+            str. The unique id of the classifier training job.
+        """
         return self._job_id
 
     @property
     def algorithm_id(self):
+        """Returns the algorithm_id of the algorithm used for generating
+            the classifier.
+
+        Returns:
+            str. The id of the algorithm used for generating the classifier.
+        """
         return self._algorithm_id
 
     @property
     def interaction_id(self):
+        """Returns the interaction_id to which the algorithm belongs.
+
+        Returns:
+            str. The id of the interaction to which the algorithm belongs.
+        """
         return self._interaction_id
 
     @property
     def exp_id(self):
+        """Returns the exploration id for which the classifier will be
+            generated.
+
+        Returns:
+            str. The id of the exploration that contains the state
+                for which classifier will be generated.
+        """
         return self._exp_id
 
     @property
     def exp_version(self):
+        """Returns the exploration version.
+
+        Returns:
+            str. The version of the exploration when the training job was
+                generated.
+        """
         return self._exp_version
 
     @property
     def next_scheduled_check_time(self):
+        """Returns the next scheduled time to check the job.
+
+        Returns:
+            datetime.datetime. The next scheduled time to check the job.
+        """
         return self._next_scheduled_check_time
 
     @property
     def state_name(self):
+        """Returns the state_name for which the classifier will be generated.
+
+        Returns:
+            str. The name of the state for which the classifier will be
+                generated.
+        """
         return self._state_name
 
     @property
     def status(self):
+        """Returns the status of the training job request.
+
+        Returns:
+            str. The status of the training job request. This can be either
+                NEW (default), PENDING (when a job has been picked up) or
+                COMPLETE.
+        """
         return self._status
 
     @property
     def training_data(self):
+        """Returns the training data used for training the classifier.
+
+        Returns:
+            list. The training data that is used for training the
```

Comment:
list(dict).

Reply:
Corrected it at other places also. Thanks!